IN-global_conf.json

```diff
@@ -1,78 +1,79 @@
 {
 	"SX1301_conf": {
 		"lorawan_public": true,
-		"clksrc": 1,
+		"clksrc": 0,
 		"clksrc_desc": "radio_1 provides clock to concentrator for most devices except MultiTech. For MultiTech set to 0.",
-		"antenna_gain": 0,
+		"antenna_gain": 1.4,
```

Comment:
Yes, I asked. It seems weird. My guess is that they use 0 in the US, and they are trying to use (in effect) 3 in India because India is EIRP; but for various reasons they adjusted everything by 4 dB in one direction and then backed off by 1 in the other. But I'm totally guessing, because the first response from them was "change this to 3dB if you have a 3dBi antenna" -- so 1.4dB seems very strange. I said "I'm going to have to defend this change in a review, I need to understand if possible"... but no response yet. (That was only yesterday afternoon.)  I'm totally OK with backing this out, because I don't understand it. If anyone reviewing has access to the Semtech 13xx docs that are under NDA, those might make this more clear. I don't have access.